app/services/data/get-last-claim-details.js

```diff
@@ -1,13 +1,22 @@
 const getClaimChildrenByIdOrLastApproved = require('./get-claim-children-by-id-or-last-approved')
 const getClaimExpenseByIdOrLastApproved = require('./get-claim-expense-by-id-or-last-approved')
 const getClaimEscortByIdOrLastApproved = require('./get-claim-escort-by-id-or-last-approved')
+const maskString = require('../helpers/mask-string')
 
-module.exports = function (reference, eligibilityId) {
+module.exports = function (reference, eligibilityId, maskChildName) {
   var result = {}
 
   return getClaimChildrenByIdOrLastApproved(reference, eligibilityId, null)
     .then(function (lastClaimChildren) {
-      result.children = lastClaimChildren
+      if (maskChildName) {
```

Comment:
Yes, all that will be returned in that case is an empty array.  The map function will not do anything with an empty array.

Also tested viewing a claim with no children and it worked as before